.env

```diff
@@ -2,4 +2,5 @@ PORT=8004
 FLASK_DEBUG=true
 SECRET_KEY=local_development_fake_key
 ENVIRONMENT=devel
-DEVEL=True
\ No newline at end of file
+DEVEL=True
+FACTORY=create_snapcraft
```

Comment:
Missing a new line!

We didn't really talk much about the new env format but I think this is too implementation specific. I think we should look at it as what makes a good readable configuration item. This currently has to accept an exact function name but what we really want is "What site should we load?"

In that sense, we could use an environment variable which is more like:
```
WEBAPP=snapcraft
```
Then the code would look for that variable and load the correct items.

Alternatively maybe it could be `BRANDSTORE=storename` and if that isn't set, it loads snapcraft by default.